data/json/items/ammo.json

```diff
@@ -2312,11 +2311,34 @@
       "id": "ammolink308",
       "copy-from": "ammolink",
       "type": "AMMO",
-      "name": "ammo belt linkage (308)",
-      "name_plural": "ammo belt linkages (308)",
+      "name": ".308 ammo belt linkage",
       "use_action": {
         "type": "ammobelt",
         "belt": "belt308"
       }
+    },
+    {
+      "id": "ammolink50",
+      "copy-from": "ammolink",
+      "type": "AMMO",
+      "name": ".50 ammo belt linkage",
+      "weight": 3,
+      "count": 50,
+      "use_action": {
+        "type": "ammobelt",
+        "belt": "belt50"
+      }
+    },
+    {
+      "id": "ammolink40mm",
```

Comment:
alphabetical ordering is required to reduce merge conflicts - especially important now multiple authors are working on these files concurrently